ctapipe/atmosphere.py

```diff
@@ -259,6 +279,12 @@ def __init__(self, table: Table):
             kind="cubic",
         )
 
+        self._height_interp = interp1d(
+            np.log10(self.table["column_density"].to("g cm-2").value),
```

Comment:
This should be `to_value(u.g / u.cm**-2)` and maybe define the unit just ones.

In general, don't use strings for units if they are easily available as module constants in astropy.

Reply:
We could also add settings for the interpolator to output nan for invalid input. I think this makes a lot of sense here, and then the calling code can decide whether having nans in the output is an error condition or not.